src/main/kotlin/spark/kotlin/Http.kt

```diff
@@ -15,30 +15,31 @@
  */
 package spark.kotlin
 
-import spark.*
+import spark.Filter
+import spark.ModelAndView
+import spark.Service
+import spark.Spark
+import spark.TemplateEngine
+import spark.TemplateViewRoute
 import kotlin.reflect.KClass
 
 // STATIC API BEGIN
-val DEFAULT_ACCEPT = "*/*"
+val DEFAULT_ACCEPT by lazy { "*/*" }
```

Comment:
What's the purpose of making it lazy?

Reply:
More over it brings contention on getter since `lazy` is thread safe.